shared/wallets/wallet/header/wallet-switcher/index.native.js

```diff
@@ -114,6 +116,9 @@ export const WalletSwitcher = (props: Props) => {
     })),
   ]
 
+  // Menu items plus pinned cancel item on bottom.
+  const height = rowHeight * (menuItems.length + 1)
```

Comment:
this was off by one, so even if you had two wallets you'd have to scroll to see the other one.